README.md

```diff
@@ -123,6 +123,24 @@ git commit -a -m "Updated translations"
 ## Styling
 We're using Nativewind, a styling system for React Native based on Tailwind CSS. Check the [Nativewind documentation](https://www.nativewind.dev/) to see what styles can be used in RN.
 
+## Creating custom icons
+
+1. Download custom icon from Figma as an SVG file.
+2. Add new icon to the iNaturalist icon set in Fontastic. Select all relevant iNaturalist icons, tap the Publish tab, and download the zip of icons.
+3. Create a glpyh file from the CSS file you just downloaded, using the following command (be sure to replace /path/to/styles with your path):
+
+```
+./node_modules/.bin/generate-icon '/path/to/styles.css' --componentName=INatIcon --fontFamily=inaturalisticons > 'src/components/INatIcon.js'
+```
+
+4. Go to `src/components/INatIcon.js` and change the name of the .ttf file to `inaturalisticons.ttf` to match the name of the .ttf file you downloaded
```

Comment:
I can't tell if this is saying to edit the line in INatIcon.js to refer to a different font file, or to change the name of the font file to `inaturalisticons.ttf`. The latter seems like it would result in less churn.